db/schema.rb

```diff
@@ -239,22 +260,4 @@
   end
 
   add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
-  add_foreign_key "attachments", "resources"
-  add_foreign_key "attributes", "resources"
-  add_foreign_key "auth_tokens", "access_codes"
-  add_foreign_key "custom_manifests", "languages"
-  add_foreign_key "custom_manifests", "resources"
-  add_foreign_key "custom_pages", "languages"
-  add_foreign_key "custom_pages", "pages"
-  add_foreign_key "follow_ups", "destinations"
-  add_foreign_key "follow_ups", "languages"
-  add_foreign_key "pages", "resources"
-  add_foreign_key "resources", "resource_types"
-  add_foreign_key "resources", "systems"
-  add_foreign_key "translated_attributes", "attributes"
-  add_foreign_key "translated_attributes", "translations"
-  add_foreign_key "translated_pages", "languages"
-  add_foreign_key "translated_pages", "resources"
-  add_foreign_key "translations", "languages"
-  add_foreign_key "translations", "resources"
```

Comment:
any reason why all the previous foreign keys are being removed?

Reply:
Oh, that's my local db, I'll revert that part